src/django/web/views.py

```diff
@@ -14,6 +16,11 @@
 
 environment['OAR_CLIENT_KEY'] = settings.OAR_CLIENT_KEY
 
+try:
+    environment['TILE_CACHE_KEY'] = Facility.current_tile_cache_key()
```

Comment:
Running `./scripts/manage migrate` is raising an exception

```
django.db.utils.ProgrammingError: relation "api_config" does not exist
LINE 1: ...fig"."created_at", "api_config"."updated_at" FROM "api_confi...
```

I think this is because `views.py` is being loaded, which causes `current_tile_cache_key` to be executed, which depends on the table's existence.

I think the solution is to move all of this code that `context` inside the `environment` method below.

Reply:
Cool, I moved all of it into the environment function in e8352f1c870176037c63e96eb202c0e5ef570450

I'm going to set up my VM from scratch again now to verify that it will build.